common/views/components/MediaObject/MediaObject.tsx

```diff
@@ -0,0 +1,89 @@
+import Image from '../Image/Image';
+import CompactCard from '../CompactCard/CompactCard';
+import { ImageType } from '../../../model/image';
+import { MediaObjectType } from '../../../model/media-object';
+import PrismicHtmlBlock from '../PrismicHtmlBlock/PrismicHtmlBlock';
+import { HTMLString } from '@weco/common/services/prismic/types';
+import styled from 'styled-components';
+import { grid, classNames, font } from '../../../utils/classnames';
+import { TextWrapperProp } from '../CompactCard/CompactCard';
+
+type Props = {
+  id: string,
+  title: string,
+  text: HTMLString | null,
+  image: ImageType,
+};
+
+type ImageWrapperProp = {
+  hasImage : boolean
+}
+
+const grid12 = grid({ s: 12, m: 12, l: 12, xl: 12 });
+
+const ImageWrapper = styled.div.attrs<ImageWrapperProp>(props => {
+  if (props.hasImage) {
+    return {
+      className: grid({ s: 2, m: 2, l: 2, xl: 2 }),
+    };
+  }
+  return {
+    className: grid12,
+  };
+})<ImageWrapperProp>``;
+
+const TextWrapper = styled.div.attrs<TextWrapperProp>(props => {
+  if (props.hasImage) {
+    return {
+      className: grid({ s: 10, m: 10, l: 10, xl: 10 }),
+    };
+  }
+  return {
+    className: grid12,
+  };
+})<TextWrapperProp>``;
+
+const TitleWrapper = styled.div.attrs(props => ({
+  className: classNames({
+    'card-link__title': true,
+    [font('wb', 4)]: true,
+  })
+}))``;
+
+export const MediaObject = ({ title, text, image }: Props) => {  
+  const ImageComponent = image && image.crops && image.crops.square && (
+    <Image {...image.crops.square} />
+  );
+  const description = text && <PrismicHtmlBlock html={text} />;
+  return (
+    <CompactCard
```

Comment:
What I was trying to convey (possibly not very well) in my point [here](https://github.com/wellcomecollection/wellcomecollection.org/pull/5650#discussion_r510022889) is that it feels back-to-front to me to have the `MediaObject` importing the `CompactCard`.

We use the term 'card' to refer to something that represents a compressed version of a piece of content available elsewhere on the site. E.g. an `EventCard`s, `ArticleCard`s, and `ExhibitionCard`s all represent individual events, articles and exhibitions respectively.

So to render a `CompactCard` inside the `MediaObject` doesn't feel great to me. For my money, a `CompactCard` is a type of `MediaObject`, so if it were able to do this the other way round, I think it would be semantically much sounder. But maybe this is something that could be left for a different PR if it's something other devs think is worth pursuing.

Reply:
I see your point. 

MediaObject is the parent and the children are Compact Card etc... 
Renaming the components would be able to solve this..and Getting any reference to compactCard now to look at MediaObject.

Reply:
Ideally I'd want the `MediaObject` to continue to be called a `MediaObject` and to be imported into the `CompactCard` I think. It can still be a component in its own right where it makes sense (like where we're using it in Prismic).

Definitely _not_ an `IconCard`, because the things that it's going to be used for on the `/covid-welcome-back` page aren't cards by our definition.

Reply:
Lets me have a quick chat with you to get some of your ideas about this , im sure i am very close to implement this.

Reply:
Thanks for that, as what we discussed 

MediaObjectBase -> Parent the mother of all motherships.
CompactCard -> inherit MediaObjectBase 
MediaObject -> inherit MediaObject 
MediaObjectList -> Calls a list of MediaObject